BEIMA.Backend.FT/BeimaClient.cs

```diff
@@ -63,6 +64,16 @@ public BeimaClient(string baseUrl)
             _httpClient.DefaultRequestHeaders.Add("Accept", "application/json");
         }
 
+        /// <summary>
+        /// Sets the authentication header of the http client to 'Bearer token'
+        /// for all outgoing http requests
+        /// </summary>
+        /// <param name="token">Jwt Token</param>
+        public void SetAuthenticationHeader(string token)
```

Comment:
I was thinking that we could even just set this in the login method itself, but this works just fine too.